agent/src/test/java/com/appland/appmap/record/RecorderTest.java

```diff
@@ -1,19 +1,17 @@
 package com.appland.appmap.record;
 
-import static com.github.stefanbirkner.systemlambda.SystemLambda.tapSystemErr;
```

Comment:
Sorry to be a nuisance about this, but looks like some more import change snuck in.